mods/flowers/init.lua

```diff
@@ -74,7 +84,14 @@ for _, m in pairs(mushrooms_datas) do
 		sunlight_propagates = true,
 		walkable = false,
 		buildable_to = true,
-		groups = {snappy=3,flammable=3,attached_node=1},
+		groups = {snappy = 3, flammable = 3, attached_node = 1},
+		drop = {
+			items = {
+				{items = {"flowers:spore_"..name},},
+				{items = {"flowers:spore_"..name}, rarity = 2,},
```

Comment:
I don't get it, it may drops twice times the spore or it's just a copy-pasta mistake ?

By the way I see multiple spore**s** in your texture. Thus it ought take a plural name.